lib/glific/contacts.ex

```diff
@@ -181,4 +181,14 @@ defmodule Glific.Contacts do
     |> Full.run(term)
     |> Repo.all()
   end
+
+  def is_new_contact(contact_id) do
+    case Glific.Messages.Message
+    |> where([c], c.contact_id == ^contact_id)
```

Comment:
@dlobo is this a right check for a new contact or I should go with optIn status. 

I wrote this function in the Contact context so that we can easily change this condition.